logger.go

```diff
@@ -102,6 +102,33 @@ func NewLoggerConfigForGCP() zap.Config {
 
 // NewDevelopmentLoggerConfig returns a new default development logger config.
 func NewDevelopmentLoggerConfig() zap.Config {
+	// from https://github.com/uber-go/zap/blob/2314926ec34c23ee21f3dd4399438469668f8097/config.go#L135
+	// but disable stacktraces, use same keys as prod, and color levels.
+	return zap.Config{
```

Comment:
this should use new debug logger but change level

Reply:
used debug config and changed level and returned the new config